hstream-processing/src/HStream/Processing/Type.hs

```diff
@@ -4,6 +4,7 @@
 module HStream.Processing.Type
   ( Timestamp,
     StreamName,
+    ConnectorName,
```

Comment:
remove this from hstream-processing and you do not need to modify any files in hstream-processing.

Reply:
done